unconvert.go

```diff
@@ -112,24 +113,24 @@ func print(name string, edits editSet) {
 	for pos := range edits {
 		fmt.Printf("%s:%d:%d: unnecessary conversion\n", pos.Filename, pos.Line, pos.Column)
 		if *flagV {
-			line := lineForOffset(buf, pos.Offset)
-			fmt.Printf("%s\n", line)
-			fmt.Printf("%s^\n", rub(line[:pos.Column-1]))
+			line := string(lineForOffset(buf, pos.Offset))
+			fmt.Println(line)
+			fmt.Println(strings.Repeat(" ", stringWidth(line[:pos.Column-1])) + "^")
 		}
 	}
 }
 
-func rub(buf []byte) []byte {
-	// TODO(mdempsky): Handle combining characters?
-	// TODO(mdempsky): Handle East Asian wide characters?
-	var res bytes.Buffer
-	for _, c := range string(buf) {
-		if !unicode.IsSpace(c) {
-			c = ' '
+func stringWidth(s string) int {
```

Comment:
What about tabs?  That's one of the reasons the original code leaves all whitespace characters alone and only converts other characters into spaces.